.github/dependabot.yml

```diff
@@ -21,5 +23,21 @@ updates:
     # Check for updates once a week
     schedule:
       interval: "weekly"
+      day: "sunday"
+      time: "22:00"
     reviewers:
       - "elastic/apm-agent-node-js"
+
+  # GitHub actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    reviewers:
+      - "elastic/observablt-ci"
```

Comment:
FWIW, I'd be happy to have this be or include "elastic/apm-agent-node-js" for review. I'm happy to "own" the GH Actions in this repo.  I don't know if there is a strong reason for the robots team to own these updates.  I don't have a strong opinion either way.

Reply:
```suggestion
      - "elastic/observablt-ci"
      - "elastic/apm-agent-node-js"
```